RELEASING.md

```diff
@@ -141,39 +141,67 @@ Tagging the Release
    ```
 7. Close the release milestone.
 
-Build Artifacts
----------------
+8. Trigger build as described in "Auto releasing using kokoro" at
+   go/grpc/java/releasing.
+
+    It runs three jobs on Kokoro, one on each platform. See their scripts:
+    `linux_artifacts.sh`, `windows.bat`, and `macos.sh`. The mvn-artifacts/
+    outputs of each script is combined into a single folder and then processed
+    by `upload_artifacts.sh`, which signs the files and uploads to Sonatype.
+
+9. Once all of the artifacts have been pushed to the staging repository, the
+   repository should have been closed by `upload_artifacts.sh`. Closing triggers
+   several sanity checks on the repository. If this completes successfully, the
+   repository can then be `released`, which will begin the process of pushing
+   the new artifacts to Maven Central (the staging repository will be destroyed
+   in the process). You can see the complete process for releasing to Maven
+   Central on the [OSSRH site](https://central.sonatype.org/pages/releasing-the-deployment.html).
+
+10. We have containers for each release to detect compatibility regressions with
+    old releases. Generate one for the new release by following the [GCR image
+    generation instructions][gcr-image]. Summary:
+    ```bash
+    # If you haven't previously configured docker:
+    gcloud auth configure-docker
+
+    # In main grpc repo, add the new version to matrix
+    ${EDITOR:-nano -w} tools/interop_matrix/client_matrix.py
+    tools/interop_matrix/create_matrix_images.py --git_checkout --release=v$MAJOR.$MINOR.$PATCH \
+        --upload_images --language java
```

Comment:
nit - long argument format using equals
```suggestion
        --upload_images --language=java
```

Reply:
All the rest of the arguments apply to language. The example has `--language cxx python ruby php`. So it isn't a simple long argument.